src/overlays/actors/ovl_Bg_Ganon_Otyuka/z_bg_ganon_otyuka.h

```diff
@@ -4,14 +4,34 @@
 #include "ultra64.h"
 #include "global.h"
 
+#define SIDE_EAST (1 << 0)
+#define SIDE_WEST (1 << 1)
+#define SIDE_SOUTH (1 << 2)
+#define SIDE_NORTH (1 << 3)
+#define SIDE_ALL (SIDE_EAST | SIDE_WEST | SIDE_SOUTH | SIDE_NORTH)
+
+typedef enum { FLASH_NONE, FLASH_GROW, FLASH_SHRINK } FlashState;
```

Comment:
Value comments are useful and ensure proper vertical formatting as a bonus, so I suggest using them here.
```suggestion
typedef enum {
    /* 0x00 */ FLASH_NONE,
    /* 0x01 */ FLASH_GROW,
    /* 0x02 */ FLASH_SHRINK
} FlashState;
```